images/dcos/bootstrap.sh

```diff
@@ -121,7 +118,15 @@ cat <<EOF >${cloud_config}
 EOF
 
 # address of the apiserver
-kube_master="http://${apiserver_host}:${apiserver_port}"
+kube_master="http://${host_ip}:${apiserver_port}"
+proxy_kube_master="http://${apiserver_host}:${proxy_apiserver_port}"
+host_kube_master="http://${host_ip}:${apiserver_port}"
```

Comment:
i guess we can't rid of this since proxy needs it so labeling it as `private` and adding a comment that ONLY the kubectl proxy should use it is probably the best we can do